integration/plugin/logging/validation_test.go

```diff
@@ -1,3 +1,5 @@
+// +build !windows
```

Comment:
Should we rename as well (`validation_linux_test.go` instead of a build-flag?

Reply:
I'm not certain. This code uses plug-ins which are not currently supported on Windows, however, if they are eventually supported, this may be possible to use directly at that time.